From e53156773a10a54961db2075fd02551e46d8fac2 Mon Sep 17 00:00:00 2001
From: Thomas BOURGOIN <thomas.bourgoin@st.com>
Date: Tue, 8 Nov 2022 14:32:18 +0100
Subject: [PATCH 0341/1141] arm64: dts: st: add crc node on stm32mp25

Add crc node for STM32MP25 platform.

Signed-off-by: Thomas BOURGOIN <thomas.bourgoin@st.com>
Change-Id: Ieb6e262f8bea01ce57eafcd1563d3d2f8d32e055
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/311509
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Tested-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 arch/arm64/boot/dts/st/stm32mp251.dtsi | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm64/boot/dts/st/stm32mp251.dtsi b/arch/arm64/boot/dts/st/stm32mp251.dtsi
index 436dde10247f..ae79d5a7edda 100644
--- a/arch/arm64/boot/dts/st/stm32mp251.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp251.dtsi
@@ -1363,6 +1363,14 @@ dcmi: dcmi@404a0000 {
 				status = "disabled";
 			};
 
+			crc: crc@404c0000 {
+				compatible = "st,stm32f7-crc";
+				reg = <0x404c0000 0x400>;
+				clocks = <&rcc CK_BUS_CRC>;
+				feature-domains = <&rifsc STM32MP25_RIFSC_CRC_ID>;
+				status = "disabled";
+			};
+
 			adc_12: adc@404e0000 {
 				compatible = "st,stm32mp25-adc-core";
 				reg = <0x404e0000 0x400>;
-- 
2.39.2

